lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -233,34 +233,33 @@
 
     // CASE: empty dashboard
     .empty-dashboard-message {
+      border: 3px solid $gray-l4;
+      background: $gray-l6;
       padding: ($baseline*2) 0;
       text-align: center;
 
       p {
         color: $lighter-base-font-color;
-        font-style: italic;
+        font-size: x-large;
         margin-bottom: $baseline;
         text-shadow: 0 1px rgba(255,255,255, 0.6);
       }
 
       a {
-        background: rgb(240,240,240);
-        @include background-image($button-bg-image);
-        background-color: $button-bg-color;
-        border: 1px solid $border-color-2;
-        border-radius: 4px;
+        background-color: $m-blue-d7;
```

Comment:
I would use one of our non m- prefixed blues here. These were used for the marketing site long ago, but shouldn't be used moving forward.